tensorboard/webapp/app_container.ng.html

```diff
@@ -19,6 +19,7 @@
 <main #main>
   <router-outlet></router-outlet>
 </main>
+<alert-snackbar></alert-snackbar>
```

Comment:
Hmm, this is not great for composability but I have no good recommendation here.